### ISSUE TITLE
transverse distal phalangeal crease(s)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11671]
Dear Peter

Hi Peter,

I hope you are doing well and that everything is going great over there!

I'm reaching out because I have a new HPO that I would love to include in the HPO database, of course still not included:

 

Label: “transverse distal phalangeal crease”.

 

Definition:

The finding is described in the literature as “transverse distal phalangeal crease(s)” or “distal phalangeal creases”. “One or more supernumerary horizontal transverse creases over the distal phalanx or fingertip pulp. These may be accompanied by prominent fingertip pads or persistent fetal finger pads”. Please, do not confound with other phlangela creases already included as a HPO term.

Cites:  

1- Orstavik et al., 2007; European Journal of Medical Genetics. 2007;50(2):155–158. PMID: 17324647. DOI: 10.1016/j.ejmg.2006.12.003.

Allanson J, Kavamura I, Neri G, Noonan J, Poss A, Kerr B. European Journal of Medical Genetics. 2007;50(6):482–483. PMID: 17974516. DOI: 10.1016/j.ejmg.2007.09.002.

 

Associated diseases: Costello syndrome, Cardiofaciocutaneous síndrome,  Noonan syndrome. Rarely in Pitt-Hopkins syndrome.

MIM 218040 COSTELLO SYNDROME

MIM PS115150- Cardiofaciocutaneous síndrome

MIM 163950 NOONAN SYNDROME 1

MIM PS610954-Pitt-Hopkins syndrome

 

ORCID- Lapunzina-

https://orcid.org/0000-0002-6324-4825

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001515
- label: Transverse distal phalangeal crease
- parents: [Abnormal fingertip morphology (HP:0001211)](https://hpo.jax.org/app/browse/term/HP:0001211)
- pmid: PMID:17324647|PMID:17974516
- def: A horizontal transverse crease over the distal phalanx or fingertip pulp. This feature may be associated with prominent fingertip pads (which are also known as persistent fetal finger pads), and if so, the crease is located distal to the fingertip pad.
- comment: Transverse distal phalangeal creases may be a feature of Costello, CFC and Noonan syndromes.
- orcid: orcid:0000-0002-6324-4825
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11671
- synonyms:
